### PR TITLE
LCBench Fix

### PR DESCRIPTION
**Pull request by wistuba**
Drop first and last epoch from the current LCBench version. Add further metrics.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: syne_tune/blackbox_repository/conversion_scripts/scripts/lcbench/lcbench.py [hunk 1]
```diff
@@ -41,7 +41,7 @@
 
 RESOURCE_ATTR = "epoch"
 
-MAX_RESOURCE_LEVEL = 52
+MAX_RESOURCE_LEVEL = 50
```
Comment (mseeger) [inline syne_tune/blackbox_repository/conversion_scripts/scripts/lcbench/lcbench.py:44]: This naming is wrong. The max_resource_level is 51, right? It was wrong before as well, I suppose.

**Review by mseeger — APPROVED** [review]
Just one little thing to fix.

File: syne_tune/blackbox_repository/conversion_scripts/scripts/lcbench/lcbench.py (hunk 1 above)
Comment (wistuba) [reply syne_tune/blackbox_repository/conversion_scripts/scripts/lcbench/lcbench.py:44]: The data has 52 epochs. Based on the notebook, the first epoch is before training started. Furthermore, they seem to discard the last epoch as well. The paper and the [repo](https://github.com/automl/LCBench) clearly state that they have 50 epochs. Therefore, I've dropped first and last data point which means we have 50 epochs in total. Does that make sense to you?

Comment (mseeger) [reply syne_tune/blackbox_repository/conversion_scripts/scripts/lcbench/lcbench.py:44]: Ah I see, you are right.

**Review by mseeger — APPROVED** [review]

**Pull request merged**